po/sv.po

```diff
@@ -7,559 +7,564 @@ msgid ""
-msgstr ""
+msgstr "<i>Konfigurera vilka kolumner som visas i översiktsvyn.</i>"
 
-#: ../src/gourmet/ui/preferenceDialog.ui.h:6
+#: ../src/gourmand/ui/preferenceDialog.ui.h:6
 msgid "Index View"
-msgstr ""
+msgstr "Översikten"
 
-#: ../src/gourmet/ui/preferenceDialog.ui.h:7
+#: ../src/gourmand/ui/preferenceDialog.ui.h:7
 msgid "<b>Card View Preferences</b>"
-msgstr ""
+msgstr "<b>Inställningar för receptkort</b>"
 
-#: ../src/gourmet/ui/preferenceDialog.ui.h:8
+#: ../src/gourmand/ui/preferenceDialog.ui.h:8
 msgid "_Allow units to change when multiplying"
-msgstr ""
+msgstr "T enheter vid mångfaldigande"
 
-#: ../src/gourmet/ui/preferenceDialog.ui.h:9
+#: ../src/gourmand/ui/preferenceDialog.ui.h:9
 msgid "_Use fractions"
-msgstr ""
+msgstr "Använd bråkform"
 
-#: ../src/gourmet/ui/preferenceDialog.ui.h:10
+#: ../src/gourmand/ui/preferenceDialog.ui.h:10
 msgid "Use fractions when possible for display"
-msgstr ""
+msgstr "Använd bråkform när det är möjligt"
 
-#: ../src/gourmet/ui/preferenceDialog.ui.h:11
+#: ../src/gourmand/ui/preferenceDialog.ui.h:11
 msgid "<i>Remove items you don't use from the Recipe Card interface.</i>"
-msgstr ""
+msgstr "<i>Dölj icke använda fält på receptkort.</i>"
 
-#: ../src/gourmet/ui/preferenceDialog.ui.h:12
+#: ../src/gourmand/ui/preferenceDialog.ui.h:12
 msgid "Card View"
 msgstr "Kortvy"
 
-#: ../src/gourmet/ui/preferenceDialog.ui.h:13
+#: ../src/gourmand/ui/preferenceDialog.ui.h:13
 msgid "<b>Shopping List Preferences</b>"
-msgstr ""
+msgstr "<b>Inställningar för inköpslista</b>"
 
-#: ../src/gourmet/ui/preferenceDialog.ui.h:14
+#: ../src/gourmand/ui/preferenceDialog.ui.h:14
 msgid ""
-"<i>What should Gourmet do with Optional Ingredients when adding recipes to "
+"<i>What should Gourmand do with Optional Ingredients when adding recipes to "
 "the shopping list?</i>"
 msgstr ""
-"<i>Vad ska Gourmet göra med valfria ingredienser när recept läggs till i "
+"<i>Vad ska Gourmand göra med valfria ingredienser när recept läggs till i "
 "inköpslistan?</i>"
 
-#: ../src/gourmet/ui/preferenceDialog.ui.h:15
+#: ../src/gourmand/ui/preferenceDialog.ui.h:15
 msgid "As_k whether to include optional ingredients"
 msgstr "Fråga om valfria ingredienser ska _ingå"
 
-#: ../src/gourmet/ui/preferenceDialog.ui.h:16
+#: ../src/gourmand/ui/preferenceDialog.ui.h:16
 msgid "_Remember user selections by default"
-msgstr ""
+msgstr "Kom ihåg användarval"
 
-#: ../src/gourmet/ui/preferenceDialog.ui.h:17
+#: ../src/gourmand/ui/preferenceDialog.ui.h:17
 msgid "_Clear remembered optional ingredients"
-msgstr ""
+msgstr "Rensa lagrade valfria ingredienser"
 
-#: ../src/gourmet/ui/preferenceDialog.ui.h:18
+#: ../src/gourmand/ui/preferenceDialog.ui.h:18
 msgid "_Always add optional ingredients"
-msgstr ""
+msgstr "Lägg alltid till valfria ingredienser"
 
-#: ../src/gourmet/ui/preferenceDialog.ui.h:19
+#: ../src/gourmand/ui/preferenceDialog.ui.h:19
 msgid "_Never add optional ingredients"
-msgstr ""
+msgstr "Lägg aldrig till valfria ingredienser"
```

Comment:
The `_` in the text marks the letter for the keyboard shortcut in Gtk.  
I suppose it would be consistent to have them in all languages.

Reply:
Yes, they should but they are a problem to translate since it is easy to get duplicates into menus.
For example the **_Go** and **_Tools** menus is currently translated to **_Visa** and **_Verktyg** which makes alt-V select one of them seemingly at random (probably a bug in Gtk), so I removed some _ when I didn't find it in the ui so I could verify everything.

I have already found some other inconsistencies in the translations so I will make a second pass and look for these :-)